### PR TITLE
docs: translate font specs file

### PR DESCRIPTION
**Pull request by picodoth**
translate font.md

### CODE REVIEW

File: docs/spec/font.en-US.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+category: Visual
+order: 3
+title: Font
+---
+
+Font is one of the most basic components of a interface design system.
```
Comment (afc163) [inline docs/spec/font.en-US.md]: It is not components. Foundation or basis will be better.

File: docs/spec/font.en-US.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Text is the major channel for users to understand application content and complete their work, and a well designed font system will greatly enhance the user's reading experience and work efficiency. The Ant Design font scheme is based on the design principle of "dynamic order" combined with the law of natural logarithm and temperament. We strongly recommend it since it has been verified by a large number of Ant products.
```
Comment (afc163) [inline docs/spec/font.en-US.md]: font scheme => typography

Comment (afc163) [reply docs/spec/font.en-US.md]: 动态秩序 => http://dorderism.org/book/dorderism.html

Comment (afc163) [reply docs/spec/font.en-US.md]: `We strongly recommend` => `We'd like to recommend`

更柔和一点

File: docs/spec/font.en-US.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+While defining the font system for a visual system, we propose to start from the following five aspects:
+
+1. font family
```
Comment (afc163) [inline docs/spec/font.en-US.md]: Should be all capitalized?

File: docs/spec/font.en-US.md (hunk 1 above)
Comment (picodoth) [reply docs/spec/font.en-US.md]: Will change to foundations.

File: docs/spec/font.en-US.md (hunk 2 above)
Comment (picodoth) [reply docs/spec/font.en-US.md]: @afc163 长知识了，不过Dorderism更像一种主义，动态秩序论，而且Dorderism是个新造的词D(ynamic)ORDERism，概念本身就不好理解，用太抽象的词别人就更不懂了吧，所以我觉得这里用dynamic order更合适。

File: docs/spec/font.en-US.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+2. Base Font Size
+3. Font Scale & Line Height
+4. Font Weight
+5. Font Color
+
+---
+
+## Font Family
+
+In order to implement a good font system, the first thing is to choose an appropriate font family. Ant Design prefers the system default font faminly and then also provides a set of alternative font libraries to maintain readability for screens on different platforms and browsers and to make sure it's always user friendly, stable and professional to end user.
```
Comment (abenhamdine) [inline docs/spec/font.en-US.md]: faminly

Comment (picodoth) [reply docs/spec/font.en-US.md]: faminly?

Comment (abenhamdine) [reply docs/spec/font.en-US.md]: I mean faminly should be family

Comment (picodoth) [reply docs/spec/font.en-US.md]: thanks, good catch!

**Comment by afc163** [comment]
image

**Comment by picodoth** [comment]
@afc163 weird, will take a look later

**Pull request merged**